### ISSUE TITLE
Statement Parsers....

### ISSUE THREAD

**Issue by jogibear9988** [bruce-dunwiddie/tsql-parser#6]
Is it planed to add more Statement Parsers?

For example, in the Select Statement, the "select" property contains only Tokens. But I'd like to find out, if it contains subselects...

Or in the From, if it contains joins. I think they should contain also statements

**Comment by bruce-dunwiddie** [comment]
Yes, it's part of the plan, but I haven't decided on a timeline.

For what you're looking for, you shouldn't technically need another parser, we just need to figure out a good way of expressing the results in the TSQLSelectStatement and/or TSQLFromClause classes and related classes, plus a few minor additions to the existing TSQLSelectStatementParser logic.

Feel free to come up with a suggested solution and submit a pull request.

**Comment by bruce-dunwiddie** [comment]
Columns are now returned in SELECT clause in latest version, https://www.nuget.org/packages/TSQL.Parser/2.0.0 .

More work still to be done to break out pieces of FROM, WHERE, and others.
